ureport/public/views.py

```diff
@@ -208,10 +208,10 @@ def render_to_response(self, context, **kwargs):
         if org.get_config('is_global'):
             location_boundaries = org.boundaries.filter(level=0)
         else:
-            state_id = self.kwargs.get('osm_id', None)
+            osm_id = self.kwargs.get('osm_id', None)
 
-            if state_id:
-                location_boundaries = org.boundaries.filter(level=2, parent__osm_id=state_id)
```

Comment:
@musamusa let's replace these magic numbers with Constants, i mean the ADMIN_LEVELS.